### PR TITLE
BUX-589: Migrate package

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
Changed name
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "projectName": "js-buxclient",
-  "projectOwner": "BuxOrg",
+  "projectName": "spv-wallet-js-client",
+  "projectOwner": "Spv Wallet",
```
Comment (kuba-4chain) [inline .all-contributorsrc:3]: This could be `SPV Wallet`

File: LICENSE [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Open BSV License
-Copyright (c) 2022 BuxOrg
+Copyright (c) 2022 Spv Wallet
```
Comment (kuba-4chain) [inline LICENSE:2]: Should be `SPV Wallet`

File: jsdoc.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Bux Javascript client",
+  "title": "Spv Wallet Javascript client",
```
Comment (kuba-4chain) [inline jsdoc.json:2]: SPV Wallet :)

File: package.json [hunk 4]
```diff
@@ -1,19 +1,19 @@
 {
-  "name": "@buxorg/js-buxclient",
+  "name": "@bitcoin-sv/spv-wallet-js-client",
   "version": "1.5.0",
-  "description": "TypeScript library for connecting to a BUX server",
+  "description": "TypeScript library for connecting to a Spv Wallet server",
```
Comment (kuba-4chain) [inline package.json:4]: SPV Wallet

File: src/index.ts [hunk 5]
```diff
@@ -130,7 +130,7 @@ class BuxClient implements TransportService {
   /**
    * Set whether to sign all requests to the server
    *
-   * This option is on (true) by default and should only be turned off in development. The Bux server needs to accept
+   * This option is on (true) by default and should only be turned off in development. The Spv Wallet server needs to accept
```
Comment (kuba-4chain) [inline src/index.ts:133]: SPV Wallet :)

File: src/index.ts [hunk 6]
```diff
@@ -168,7 +168,7 @@ class BuxClient implements TransportService {
   }
 
   /**
-   * Admin only: Get stats about the Bux server
+   * Admin only: Get stats about the Spv Wallet server
```
Comment (kuba-4chain) [inline src/index.ts:171]: SPV Wallet :)

File: src/index.ts [hunk 7]
```diff
@@ -449,7 +449,7 @@ class BuxClient implements TransportService {
   /**
    * Revoke an access key and invalidate in the database
    *
-   * After this function is successfully called, the access key cannot be used anymore on a Bux server
+   * After this function is successfully called, the access key cannot be used anymore on a Spv Wallet server
```
Comment (kuba-4chain) [inline src/index.ts:452]: SPV Wallet :)

File: src/index.ts [hunk 8]
```diff
@@ -515,7 +515,7 @@ class BuxClient implements TransportService {
    * Create a new destination to receive bsv with and return that destination
    *
    * This function allows you to create a destination that will be monitored on-chain and will import any transactions
-   * related to that destination into Bux. This is legacy functionality in Bitcoin and should only be used if a p2p
+   * related to that destination into Spv Wallet. This is legacy functionality in Bitcoin and should only be used if a p2p
```
Comment (kuba-4chain) [inline src/index.ts:518]: SPV Wallet :)

File: src/index.ts [hunk 9]
```diff
@@ -667,7 +667,7 @@ class BuxClient implements TransportService {
    * @see {@link SendToRecipients}
    * @param {Recipients} recipients A list of recipients and a satoshi value to send to them
    * @param {Metadata} metadata     Key value object to use to add to the draft transaction
-   * @return {DraftTransaction}     Complete draft transaction object from Bux, all configuration options filled in
+   * @return {DraftTransaction}     Complete draft transaction object from Spv Wallet, all configuration options filled in
```
Comment (kuba-4chain) [inline src/index.ts:670]: SPV Wallet :)

File: src/index.ts [hunk 10]
```diff
@@ -678,7 +678,7 @@ class BuxClient implements TransportService {
    *
    * @param {TransactionConfigInput} transactionConfig Configuration of the new transaction
    * @param {Metadata} metadata                        Key value object to use to add to the draft transaction
-   * @return {DraftTransaction}                        Complete draft transaction object from Bux, all configuration options filled in
+   * @return {DraftTransaction}                        Complete draft transaction object from Spv Wallet, all configuration options filled in
```
Comment (kuba-4chain) [inline src/index.ts:681]: SPV Wallet :)

File: src/index.ts [hunk 11]
```diff
@@ -769,15 +769,15 @@ class BuxClient implements TransportService {
   }
 
   /**
-   * Record a Bitcoin transaction (in hex) into Bux
+   * Record a Bitcoin transaction (in hex) into Spv Wallet
```
Comment (kuba-4chain) [inline src/index.ts:772]: SPV Wallet :)

File: src/index.ts [hunk 12]
```diff
@@ -769,15 +769,15 @@ class BuxClient implements TransportService {
...
    *
-   * This will only work of an input or output of the transaction can be related to a Bux user. Bux does not record
+   * This will only work of an input or output of the transaction can be related to a Spv Wallet user. Spv Wallet does not record
```
Comment (kuba-4chain) [inline src/index.ts:774]: SPV Wallet :)

File: src/index.ts [hunk 13]
```diff
@@ -769,15 +769,15 @@ class BuxClient implements TransportService {
...
    * unrelated transaction into its database.
    *
    * @param {string} hex         Hex string of the Bitcoin transaction
    * @param {string} referenceID Optional reference ID (draft transaction ID)
    * @param {Metadata} metadata  Key value object to use to add to the transaction
-   * @return {Transaction}       The Bux transaction object
+   * @return {Transaction}       The Spv Wallet transaction object
```
Comment (kuba-4chain) [inline src/index.ts:780]: SPV Wallet :)

File: src/index.ts [hunk 14]
```diff
@@ -797,7 +797,7 @@ class BuxClient implements TransportService {
   }
 
   /**
-   * Admin only: Register a new xPub into the Bux server
+   * Admin only: Register a new xPub into the Spv Wallet server
```
Comment (kuba-4chain) [inline src/index.ts:800]: SPV Wallet :)

File: src/interface.ts [hunk 15]
```diff
@@ -71,13 +71,13 @@ export interface XPub {
   /**
    * Next internal (change address) number to use for a new destination
    *
-   * NOTE: Do not use this to create new destinations, always let Bux create the destination
+   * NOTE: Do not use this to create new destinations, always let Spv Wallet create the destination
```
Comment (kuba-4chain) [inline src/interface.ts:74]: SPV Wallet :)

File: src/interface.ts [hunk 16]
```diff
@@ -71,13 +71,13 @@ export interface XPub {
...
    */
   next_internal_num: number;
   /**
    * Next external number to use for a new destination
    *
-   * NOTE: Do not use this to create new destinations, always let Bux create the destination
+   * NOTE: Do not use this to create new destinations, always let Spv Wallet create the destination
```
Comment (kuba-4chain) [inline src/interface.ts:80]: SPV Wallet :)

File: src/interface.ts [hunk 17]
```diff
@@ -638,7 +638,7 @@ export interface PubKey {
 
 
 /**
- * Client options for instantiating a new Bux client
+ * Client options for instantiating a new Spv Wallet client
```
Comment (kuba-4chain) [inline src/interface.ts:641]: SPV Wallet :)

File: .all-contributorsrc (hunk 1 above)
Comment (Nazarii-4chain) [reply .all-contributorsrc]: Updated names

File: src/index.ts [hunk 18]
```diff
@@ -247,7 +247,7 @@ class BuxClient implements TransportService {
   /**
    * Admin only: Get a paymail by address
    *
-   * @param address string Paymail address (ie. example@bux.org)
+   * @param address string Paymail address (i.e. example@spv-wallet.org)
```
Comment (chris-4chain) [inline src/index.ts:250]: I racall that we established that a domain should be `example.com` e.g "alias@example.com"

File: src/index.ts [hunk 19]
```diff
@@ -281,7 +281,7 @@ class BuxClient implements TransportService {
    * Admin only: Create a new paymail for an xPub ID
    *
    * @param {string} xpub_id The ID of the xPub to register the paymail to (note: NOT the xpub itself, but the ID (hash))
-   * @param {string} address Paymail address (ie. example@bux.org)
+   * @param {string} address Paymail address (i.e. example@spv-wallet.org)
```
Comment (chris-4chain) [inline src/index.ts:284]: Same as above: `example.com`

File: src/index.ts [hunk 20]
```diff
@@ -293,7 +293,7 @@ class BuxClient implements TransportService {
   /**
    * Admin only: Delete a paymail
    *
-   * @param address string Paymail address (ie. example@bux.org)
+   * @param address string Paymail address (i.e. example@spv-wallet.org)
```
Comment (chris-4chain) [inline src/index.ts:296]: Same as above: `example.com`

**Review by kuba-4chain — APPROVED** [review]

**Pull request merged**